gulpfile.js

```diff
@@ -4,6 +4,7 @@ var gulp = require('gulp');
 var yargs = require('yargs');
 var mocha = require('gulp-mocha');
 var jshint = require('gulp-jshint');
+require('coffee-script/register');
```

Comment:
What is this for?  Just wanted to make sure it's needed since it looks like an orphaned `require` statement.